### PR TITLE
Fix a lot of typos

### PR DESCRIPTION
**Pull request by nikosavola**
This PR fixes a lot of typos, mostly in comments, some in function and variable names.

The typos were scoped automatically with [scspell](https://pypi.org/project/scspell/)

### CODE REVIEW

File: skrf/media/media.py [hunk 1]
```diff
@@ -950,7 +950,7 @@ def shunt_capacitor(self,C,*args,**kwargs):
         Returns
         --------
         shunt_capacitor : :class:`~skrf.network.Network` object
-                shunted capcitor(2-port)
+                shunted capaciton(2-port)
```
Comment (jhillairet) [inline skrf/media/media.py:953]:
```suggestion
                shunted capacitor (2-port)
```

**Review by jhillairet — APPROVED** [review]
Very nice, thank you. I've only spotted 1 thing. 

And the winner is... "characteristic" ;)

**Review by Vinc0110 — APPROVED** [review]
Wow, that's a lot of typos... Thanks for fixing them.

Just an idea for further work on this:
It would probably be even more important and useful to do this kind of check on the Jupyter notebooks for the examples, tutorials and API documentation in `scikit-rf/doc/source/`. That should be possible, right?

**Comment by nikosavola** [comment]
> Wow, that's a lot of typos... Thanks for fixing them.
> 
> Just an idea for further work on this:
> It would probably be even more important and useful to do this kind of check on the Jupyter notebooks for the examples, tutorials and API documentation in `scikit-rf/doc/source/`. That should be possible, right?

Yes, should be similarly possible. I might even do that at some point.

**Comment by jhillairet** [comment]
> Yes, should be similarly possible. I might even do that at some point.

OK. I will merge for now. Thank you !

**Pull request merged**